test/gurka/test_turn_lanes.cc

```diff
@@ -49,15 +50,28 @@ TEST(Standalone, TurnLanes) {
                                                 DirectionsLeg_Maneuver_Type_kDestination});
   gurka::assert::raw::expect_path_length(result, 0.7, 0.001);
 
-  auto maneuver = result.directions().routes(0).legs(0).maneuver(1);
-  EXPECT_EQ(maneuver.type(), DirectionsLeg_Maneuver_Type_kRight);
+  validate_turn_lanes(result, {
+                                  {3, "[ left | through | through;*right* ACTIVE ]"},
+                                  {4, "[ left | through | through | right ]"},
```

Comment:
Why are none of the lanes active here?

Reply:
@danpaz This is because we are on B----C and no left and right exists in the ascii map.  Lanes are tossed in the guidance code if they don't match the graph.

Reply:
Wouldn't the `through` be active in that case? /cc @dgearhart maybe I'm misremembering here.

Reply:
@danpaz @gknisely I do not think we picked a turn lane for destination in the `GetExpectedTurnLaneDirection` method